File: SampleApp/java/app/src/main/java/jp/fluct/sample/samplefluctsdkapp/InfeedActivity.java
package jp.fluct.sample.samplefluctsdkapp;

import android.os.Bundle;
import android.support.v4.app.FragmentTransaction;
import android.support.v7.app.AppCompatActivity;

public class InfeedActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_infeed);

        if (savedInstanceState == null) {
            ContentsListFragment fragment = new ContentsListFragment();
            FragmentTransaction ft = getSupportFragmentManager().beginTransaction();
            ft.add(R.id.activity_infeed, fragment);
            ft.commit();
        }
    }
}

File: SampleApp/kotlin/app/src/main/java/jp/fluct/sample/kotlinapp/InfeedActivity.kt
package jp.fluct.sample.kotlinapp

import android.os.Bundle
import android.support.v7.app.AppCompatActivity

class InfeedActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_infeed)

        if (savedInstanceState == null) {
            val fragment = ContentsListFragment()
            val ft = supportFragmentManager.beginTransaction()
            ft.add(R.id.activity_infeed, fragment)
            ft.commit()
        }
    }
}